### DISCUSSION TITLE
Noob has questions

### DISCUSSION THREAD

**Discussion by gamerscircle** [pivpn/pivpn#1813] [Q&A]
I am pretty sure that I get the understanding on how to setup/install it.   I do understand the purpose of a VPN, but what I can't seem to figure out.  

If I setup the server and do the portforward, that part is only for those that need to have access from the WAN side, correct?  On the LAN side, are all devices connected through the vpn or only if they use the Wireguard client and have a license?

if is possible to have one or two clients that can use the VPN server for out going, but all other clients are not subjected to the VPN server?

I sure hope this makes sense, thank you

**Comment by orazioedoardo** [comment]
> If I setup the server and do the portforward, that part is only for those that need to have access from the WAN side, correct?

Yes.

> On the LAN side, are all devices connected through the vpn

On the LAN side you are not supposed to connect to the server at all.

PiVPN is a tool to easily create a VPN server in your location so you can access that location network **remotely** and safely from the outside while also tunnelling internet traffic through it.

**Reply by gamerscircle** [reply]
I appreciate your time in explaining this to me, I was looking for a solution that would allow me to be at home [on the LAN side] and have my own VPN connection going out.  Thank you again.

**Reply by orazioedoardo** [reply]
Then you need to install PiVPN at the remote location when you want the VPN to be and copy configurations to clients in your LAN or on the router itself.

**Comment by gamerscircle** [comment]
okay, I will have to think of another option then, thank you again!
